cli/plasmo/src/features/extension-devtools/plasmo-extension-manifest.ts

```diff
@@ -193,11 +192,11 @@ export class PlasmoExtensionManifest {
         )
       }
 
-      const contentScript = {
+      const contentScript = this.#injectEnv({
```

Comment:
Why doesn't `getOverrideManifest` do this for us automatically?

Reply:
Override manifest only concerns itself with the manifest in packagejson, meanwhile this focuses on the config exported by contentscripts